lib/osf-components/addon/components/search-result-card/registration-secondary-metadata/template.hbs

```diff
@@ -4,7 +4,7 @@
         <dd>{{@result.description}}</dd>
     {{/if}}
     {{#if @result.provider}}
-        <dt>{{t 'osf-components.search-result-card.preprint_provider'}}</dt>
```

Comment:
Dang, what goofball copy-pasted this? Purely hypothetical question, no need to blame here 😅